news/2020-19-11-mappers-report-september-october.md

```diff
@@ -0,0 +1,211 @@
+
+Another map to highlight is his rendition of [Fallujah - Venom Upon the Blade](https://osu.ppy.sh/beatmapsets/1176016#taiko/2452862), another well crafted streamy 240BPM map with a really tough outro that's sure to bring a fun challenge to anyone capable of bashing drums at this level!
+
+I speak for many people when I say we are looking forward to seeing Heaxys develop as a mapper and bring more exquisite metal maps to the ranked section!
+
+### gaston_2199 (osu!taiko)
+
+While definitely not a new face to the ranked section, [gaston_2199](https://osu.ppy.sh/users/5938161) is a highly underrated mapper whose name is rarely brought up in conversations, even though they truly deserve it! With [their first ranked map](https://osu.ppy.sh/beatmapsets/519474#taiko/1103553) ranked back in 2016, where they got credited for their nice variations in such a variety-lacking drum'n'bass song!
+
+Not only does gaston_2199 like to map DnB, but they also recently got their hands onto something more technical. [Onis Keyboard](https://osu.ppy.sh/beatmapsets/1176086#taiko/2453006) by the artist pm04034 is a pretty big set, offering 7 difficulties for players of all levels. The top diff being especially interesting, due to the nature of the song, which offers many rhythmic changes and a bit variety of snappings that all got implemented in a very clean style.
+
+The volume control is also something that gaston_2199 never misses to play around with, which is something that is often overlooked. Complex snaps, nicely playable and some SV changes every now and then form a very engaging and fun to play map! We hope we never have to miss gaston_2199 in the ranked section, as they keep a very high quality throughout all the years of their activity. Leave them a nice comment and go enjoy their maps if you haven't already!
+
+### -MomoX (osu!catch)
+
+[-MomoX](https://osu.ppy.sh/users/2150415) is a promising osu!catch mapper in the Philippines. Starting off his mapping career since 2018, -MomoX has a solid understanding of mapping concepts, techniques and musical knowledge. As a high-tier fruit catcher, -MomoX has been incorporating innovative patterns and difficulty settings into his renditions to test on players' reading, speed and stamina.
+
+In terms of mapping style, -MomoX is a slider enthusiast who loves to emphasize strong beats by hyperdash sliders. He is capable of maneuvering such gimmicks consistently without breaking the flow. Prime example would be [rejection - Signal feat. Such](https://osu.ppy.sh/beatmapsets/1129636#fruits/2359821), his recently ranked marathon. Hereby I also introduce his slider hell map [Halozy - Kikoku Doukoku Jigokuraku](https://osu.ppy.sh/beatmapsets/990511#fruits/2071833), which features a flurry of droplets by using high slider tick rate to design odd-shaped sliders.
```

Comment:
```suggestion
In terms of mapping style, -MomoX is a slider enthusiast who loves to emphasize strong beats via hyperdash sliders. He is capable of maneuvering such gimmicks consistently without breaking the flow. A prime example would be [rejection - Signal feat. Such](https://osu.ppy.sh/beatmapsets/1129636#fruits/2359821), his recently ranked marathon. Hereby I also introduce his slider hell map [Halozy - Kikoku Doukoku Jigokuraku](https://osu.ppy.sh/beatmapsets/990511#fruits/2071833), which features a flurry of droplets by using a high slider tick rate to design odd-shaped sliders.
```
not sure but 'via' sounded better to me
seems like an article is needed for those two